templates/slurm.txt

```diff
@@ -296,6 +297,12 @@ Order = 10
         DefaultValue = Standard_F2s_v2
         Config.MultiSelect = true
 
+        [[[parameter NodeTags]]]
+        Label = VM Tags
+        Description = Tags applied to all nodes
+        ParameterType = Record
+        DefaultValue := {}
```

Comment:
The default value should be := []
:= {} is a list
:= [] is a dict

It is unlike most syntaxes